onnxruntime/core/providers/coreml/builders/impl/base_op_builder.cc

```diff
@@ -110,7 +110,8 @@ bool BaseOpBuilder::IsInputDtypeSupport(const Node& node, size_t idx,
   }
 
   // float is supported
-  if (input_type == ONNX_NAMESPACE::TensorProto_DataType_FLOAT) {
+  if (input_type == ONNX_NAMESPACE::TensorProto_DataType_FLOAT ||
+      input_type == ONNX_NAMESPACE::TensorProto_DataType_INT64) {
```

Comment:
it seems like a big change to enable int64 by default now. should we start by enabling it per op builder instead? e.g., by overriding `HasSupportedInputsImpl()`

Reply:
I think it becomes a much larger change to do operator by operator for a model where the input data is int64 (vs vision models where the input is float)

By enabling it here, the ORT unit tests should run and we should have good coverage with int64 data shouldn't we?

Reply:
ok. I meant a big change in behavior vs a big code change.

> By enabling it here, the ORT unit tests should run and we should have good coverage with int64 data shouldn't we?

hopefully this is the case.